lib/plugins/setedit/README

```diff
@@ -0,0 +1,20 @@
+setedit Plugin for DokuWiki
+
+Includes file upload into wikipages depending on the user rights.
+
+If you install this plugin manually, make sure it is installed in
+lib/plugins/setedit/ - if the folder is called different it
+will not work!
+
+----
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; version 2 of the License
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+See the COPYING file in your DokuWiki folder for details
```

Comment:
@yvonnel098 We are using the MIT license rather than GPL. You can get it here: https://github.com/unfoldingWord-dev/Door43/blob/master/lib/plugins/door43obs/LICENSE